server/liveblog/validator.py

```diff
@@ -89,3 +89,51 @@ def _validate_httpsurl(self, httpsurl, field, value):
                             response.status_code
                         ),
                     )
+
+    def _validate_settings(self, settings):
+        errors = []
+        required_schema = {
+            "features": {
+                "syndication": {"type": bool, "required": True},
+                "marketplace": {"type": bool, "required": True},
+            },
+            "limits": {
+                "blogs": {"type": int, "min": 1, "required": True},
+                "themes": {"type": int, "min": 1, "required": True},
+                "blog_members": {"type": int, "min": 1, "required": True},
+            },
+        }
+
+        for key, config in settings.items():
+            if key == "network":
+                # Special case for 'network' plan
+                if not isinstance(config, int) or config < 1:
+                    errors.append(
+                        f"The value for '{key}' must be an integer and at least 1."
+                    )
+            else:
```

Comment:
Maybe we can remove this one out and we can take care of it in the features service that I'm currently working on. Thoughts?

Reply:
Hmm, I think we still need it here since the 'network' value is just an int and not a dict like the other plans so the validation will fail if we remove this part and leave the rest logic that checks the dicts. Unless of course we change network to also be a dict with "features" and "limits". 

What do you think ?

Reply:
yeah but my thinking was more about getting rid of network from the settings and only returning "true" when checking limitations if the current subscription level is network, even without checking the settings. That way we could avoid some complexity here.

Reply:
Oh, so essentially the config will have all other plans except network then on the features service it handles that case there. Makes sense